data/adoptions.yaml

```
@@ -1,8 +1,6 @@
 main:
-- url: https://docs.automotivelinux.org/docs/en/master/architecture/reference/security/part-8/1-FOTA.html
+- url: https://docs.automotivelinux.org/en/pike/#03_Architecture_Guides/02_Security_Blueprint/09_Update_%28Over_The_Air%29/#_top
```

Nit: do we need to include the `_top` fragment?
```suggestion
- url: https://docs.automotivelinux.org/en/pike/#03_Architecture_Guides/02_Security_Blueprint/09_Update_%28Over_The_Air%29/
```